lib/net.c

```diff
@@ -1365,6 +1373,55 @@ setup_ipv6_addrs(struct gatekeeper_if *iface)
 	}
 }
 
+static int
+check_port_rss_key_update(struct gatekeeper_if *iface, uint16_t port_id)
+{
+	uint8_t rss_hash_key[GATEKEEPER_RSS_KEY_LEN];
+	struct rte_eth_rss_conf rss_conf = {
+		.rss_key = rss_hash_key,
+		.rss_key_len = GATEKEEPER_RSS_KEY_LEN,
+	};
+	int ret = rte_eth_dev_rss_hash_conf_get(port_id, &rss_conf);
+	if (ret != 0) {
+		switch (ret) {
```

Comment:
Style: add `case 0:` with a single `break;` to avoid the `if` statement and unindent the `switch`. You'll need to replace the `break;` of the other cases with `return ret;`.